server/src/api/read-projects.ts

```diff
@@ -40,6 +20,7 @@ function readArchitect(project: string, basedir: string, architect: any) {
       ? Object.keys(value.configurations).map(name => ({ name }))
       : [];
     return {
+      description: '',
```

Comment:
The issue with the description field is a good catch. But I think the proper fix is to change the schema to remove the description field. It's actually not available in angular.json

Reply:
I also removed resolver for `description` field.